react-components/src/architecture/concrete/pointsOfInterest/ads/PointsOfInterestAdsProvider.ts

```diff
@@ -181,6 +191,14 @@ function poiItemToInstance(item: PoiItem): PointsOfInterestInstance<ExternalId>
       sceneState: item.sceneState
     }
   };
+
+  async function getUserName(ownerId: string, sdk: CogniteClient): Promise<string | undefined> {
+    const profiles = await sdk.profiles.retrieve([{ userIdentifier: ownerId }]);
+
+    const name = profiles.filter(isDefined).map((profile) => profile.displayName ?? undefined);
+
+    return name[0];
+  }
```

Comment:
Just a mental note that we should merge this with the code we use for PoI comment name translation